exercises/shared/.docs/debug.md

```diff
@@ -0,0 +1,9 @@
+# Debugging
+
+To help with debugging, you can use the fact that all console output will be captured and shown in the test results window. You can output any value by passing it to `IO.inspect`:
+
+```elixir
+"hello"
+|> String.upcase()
+|> IO.inspect()
```

Comment:
I think this is the best example of how `IO.inspect` gets used in the wild, but it's not the simplest. Would it be better to give a simpler example, without pipes or the `String` module?

Reply:
or maybe put the inspect between two functions and comment how it allows us to introspect the intermediate value?